src/CalculatorUnitTests/DateCalculatorUnitTests.cpp

```diff
@@ -494,6 +494,31 @@ namespace DateCalculationUnitTests
             }
         }
 
+        TEST_METHOD(DateCalcViewModelDateDiffSignTest)
+        {
+            auto viewModel = ref new DateCalculatorViewModel();
+
+            viewModel->IsDateDiffMode = true;
+            VERIFY_IS_TRUE(viewModel->IsDateDiffMode);
+
+            viewModel->FromDate = DateUtils::SystemTimeToDateTime(date[10]);
+            viewModel->ToDate = DateUtils::SystemTimeToDateTime(date[6]);
+
+            auto dfd = viewModel->StrDateDiffResultInDays;
+            auto dfffg = viewModel->StrDateDiffResult;
+            //// check negative difference
+            VERIFY_IS_FALSE(viewModel->IsDiffInDays);
+            VERIFY_ARE_EQUAL(StringReference(L"305 days"), viewModel->StrDateDiffResultInDays);
+            VERIFY_ARE_EQUAL(StringReference(L"10 months"), viewModel->StrDateDiffResult);
+
+            //// check positive difference
+            viewModel->FromDate = DateUtils::SystemTimeToDateTime(date[6]);
+            viewModel->ToDate = DateUtils::SystemTimeToDateTime(date[10]);
+            VERIFY_IS_FALSE(viewModel->IsDiffInDays);
+            VERIFY_ARE_EQUAL(StringReference(L"- 305 days"), viewModel->StrDateDiffResultInDays);
```

Comment:
Are these tests still valid?

Reply:
You are right, I forgot to remove them!

Reply:
It can still be interesting to test when FromDate < ToDate, I will keep them but modify expected results